tests/gradients/core/test_pulse_gradient.py

```diff
@@ -43,23 +43,75 @@ class TestSplitEvolOps:
     ham_single_q_pwc = lambda _: qml.pulse.pwc((2.0, 4.0)) * qml.PauliZ(0)
     ham_two_q_pwc = lambda _: qml.pulse.pwc((2.0, 4.0)) * (qml.PauliZ(0) @ qml.PauliX(1))
 
-    split_evol_ops_test_cases = [
-        (ham_single_q_const, [0.3], 2.3, qml.PauliX(0)),
-        (ham_single_q_const, [0.3], 2.3, qml.Hamiltonian([0.2], [qml.PauliZ(0)])),
-        (ham_single_q_const, [0.3], 2.3, 1.2 * qml.PauliZ(0)),
-        (ham_single_q_const, [0.3], 2.3, qml.dot([1.9], [qml.PauliZ(0)])),
-        (ham_single_q_pwc, [np.linspace(0, 1, 13)], (0.6, 1.2), qml.PauliY(1)),
-        (
-            ham_single_q_pwc,
-            [np.linspace(0, 1, 13)],
-            (0.6, 1.2),
-            0.2 * qml.PauliX(1) + 0.9 * qml.PauliZ(1),
-        ),
+    split_evol_ops_test_cases_pauliword = [
+        (ham_single_q_const, [0.3], 2.3, qml.PauliX(0), "X"),
+        (ham_single_q_pwc, [np.linspace(0, 1, 13)], (0.6, 1.2), qml.PauliY(1), "Y"),
         (
             ham_two_q_pwc,
             [np.linspace(0, 1, 13)],
             (0.2, 0.6, 0.9, 1.8),
             qml.PauliY(0) @ qml.PauliX(1),
+            "YX",
+        ),
+        (ham_single_q_const, [0.3], 2.3, qml.Hamiltonian([0.2], [qml.PauliZ(0)]), "Z"),
+        (ham_single_q_const, [0.3], 2.3, 1.2 * qml.PauliZ(0), "Z"),
+        (ham_single_q_const, [0.3], 2.3, qml.dot([1.9], [qml.PauliZ(0)]), "Z"),
```

Comment:
Would it make sense to also add explicit cases with `qml.s_prod` and `qml.sum`, in case the default behavior for ops with `*` and `+` changes?

Reply:
Sure, can do :)